src/cogite/plugins.py

```diff
@@ -16,8 +21,8 @@ def get_extra_commands():
 
 
 def _get_plugins(namespace):
-    entry_points = pkg_resources.iter_entry_points(namespace)
-    return [entry_point.resolve() for entry_point in entry_points]
+    entry_points = importlib_metadata.entry_points(group=namespace)  # pylint: disable=unexpected-keyword-arg
```

Comment:
Ça sent le code code pas testé, parce que je crois bien que pylint a raison : `entry_points()` ne prend pas d'argument. :)

Apparemment il faudrait écrire un truc comme ça : 

```suggestion
    entry_points = importlib_metadata.entry_points().get(group=namespace, ())
```

Reply:
Homme de peu de foi !
https://github.com/python/cpython/blob/main/Lib/importlib/metadata/__init__.py#L957
https://github.com/python/importlib_metadata/blob/main/importlib_metadata/__init__.py#L959

Reply:
Les arguments n'ont été ajoutés que dans Python 3.10 : https://github.com/python/cpython/commit/f917efccf8d5aa2b8315d2a832a520339e668187. Hipster ! :)

Reply:
Ok, je m'étais contenté de tester avec le backport `importlib_metadata` :confounded: